packages/jest-cli/src/cli/runCLI.js

```diff
@@ -45,6 +45,17 @@ module.exports = (
     return;
   }
 
+  const config = readConfig(argv, root);
+  if (argv.showConfig) {
+    const output = ({config}) => {
+      pipe.write(JSON.stringify(config, null, 2));
```

Comment:
Ah, I didn't see that right below inside of `_run`. I agree, I'll get that moved.